modules/imgcodecs/test/test_grfmt.cpp

```diff
@@ -87,11 +87,17 @@ const string all_images[] =
     "readwrite/uint16-mono2.dcm",
     "readwrite/uint8-rgb.dcm",
 #endif
+#if defined(HAVE_PNG) || defined(HAVE_SPNG)
```

Comment:
We have GDAL library integration which should support all these image types (PNG, JPEG, TIFF). Perhaps it should be accounted too, though I didn't try to test GDAL backend specifically (with no other backends).